lib/Importer/ONNXModelLoader.cpp

```diff
@@ -3814,6 +3814,17 @@ Error ONNXModelLoader::loadMFCC(const ONNX_NAMESPACE::NodeProto &op,
   return Error::success();
 }
 
+Error ONNXModelLoader::loadSign(const ONNX_NAMESPACE::NodeProto &op,
+                                const ArgumentDictionaryTy &dict) {
+  const std::string &opName = loadOperatorName(op);
+  NodeValue in;
+  ASSIGN_VALUE_OR_RETURN_ERR(in, getNodeValueByName(op.input(0)));
+  auto outTy = mod_.uniqueType(*(in.getType()));
+  Node *node = G_->createSign(opName, outTy, in);
```

Comment:
We're always using the same input and output type, so seems reasonable to just use the API that does that:
```suggestion
  Node *node = G_->createSign(opName, in);
```

Reply:
Done